test/runtests.jl

```diff
@@ -45,6 +46,53 @@ function test_truth_most_likely_with_wise_psf()
     @test good_ll > bad_ll
 end
 
+function test_real_bright(band_id::Int64)
```

Comment:
Stating what you want the test to test in the function name, in a quasi sentence, is considered good practice.... Helps when a test starts failing